lib/vsc/install/shared_setup.py

```diff
@@ -24,19 +24,46 @@
 import shutil
 import sys
 import re
+
+try:
+    from unittest.suite import TestSuite
+except ImportError:
+    from unittest import TestSuite
+
 from distutils import log  # also for setuptools
 from distutils.dir_util import remove_tree
 
+from setuptools.command.test import test as TestCommand
+from setuptools.command.test import ScanningLoader
```

Comment:
What if `setuptools` is not available? It's not a part of the standard Python library...